tests/State/ScopeTest.php

```diff
@@ -29,20 +29,15 @@ public function testSetTag(): void
         $this->assertSame(['foo' => 'bar', 'bar' => 'baz'], $event->getTagsContext()->toArray());
     }
 
-    public function setTags(): void
+    public function testSetTags(): void
     {
         $scope = new Scope();
         $scope->setTags(['foo' => 'bar']);
-
```

Comment:
Please revert this and the subsequent changes on the blank line as they improve the readability and simplifies the code review

Reply:
8e26743